pkg/controller/backup_controller_test.go

```diff
@@ -231,6 +235,23 @@ func TestProcessBackup(t *testing.T) {
 				res.Status.Expiration.Time = expiration
 				res.Status.Phase = v1.BackupPhase(phase)
 
+				// We don't care about the value of the timestamps here, just whether or
+				// not they're present in the patchMap.
+				// If there's an error, it's mostly likely that the key wasn't found
```

Comment:
nit: `// If there's an error, it's most likely that the key wasn't found`

Reply:
Fixed